dbms/src/Storages/Page/V3/PageDirectory.cpp

```diff
@@ -831,7 +831,7 @@ PageIdV3Internal PageDirectory::getNormalPageId(PageIdV3Internal page_id, const
                 }
                 else
                 {
-                    return PageIdV3Internal(0, INVALID_PAGE_ID);
+                    return PageIdV3Internal(INVALID_PAGE_ID, 0);
```

Comment:
so the first num is stored in lower 64 bit and the second num is higher 64 bit?

Reply:
yes, 
```
UInt128(const uint64_t low, const uint64_t high)
low is page_id
high is namespace_id
```

Reply:
Maybe use `buildV3Id` to generate `PageIdV3Internal`?